docs/protocol-methods.rst

```diff
@@ -588,6 +590,12 @@ as an input (spends it).
 
     The output index, a non-negative integer. (sometimes called prevout_n, in inputs)
 
+  *spk_hint*
+
+    The scriptPubKey (output script) corresponding to the outpoint, as a hexadecimal
+    string. This is optional, and if provided might be used by the server to find
+    the outpoint. The behaviour is undefined if an incorrect value is provided.
```

Comment:
Maybe worth mentioning that some (personal) server implementations might consider this required?

Reply:
done